src/request-pipeline/context.js

```diff
@@ -172,8 +172,10 @@ export default class RequestPipelineContext {
         var isRedirect              = this.destRes.headers['location'] &&
                                       REDIRECT_STATUS_CODES.indexOf(this.destRes.statusCode) > -1;
         var requireAssetsProcessing = (isCSS || isScript || isManifest) && this.destRes.statusCode !== 204;
+        var isNotModified           = this.req.method === 'GET' && this.destRes.statusCode === 304 &&
+                                      (this.req.headers['if-modified-since'] || this.req.headers['if-none-match']);
```

Comment:
@LavrovArtem @churkin 
For consistency, property definition should be moved to constructor and property calcualation to the `_initRequestNatureInfo`.